### PR TITLE
Cleaning up couple of lint errors and warnings

### PR DESCRIPTION
**Pull request by pprucia**
#### APK
[Download build #8731](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8731/artifact/build/artifact/wire-dev-PR757-8731.apk)
[Download build #8737](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8737/artifact/build/artifact/wire-dev-PR757-8737.apk)

### CODE REVIEW

**Comment by AndroidBob** [comment]
Can one of the admins verify this patch?

**Comment by hakonbo** [comment]
build again

File: app/src/main/java/com/waz/zclient/pages/main/conversationpager/SlidingPaneLayout.java [hunk 1]
```diff
@@ -1021,30 +1014,7 @@ protected boolean drawChild(Canvas canvas, View child, long drawingTime) {
...
-                }
-            } else {
-                if (child.isDrawingCacheEnabled()) {
-                    child.setDrawingCacheEnabled(false);
-                }
-                result = super.drawChild(canvas, child, drawingTime);
-            }
-        }
-
+        result = super.drawChild(canvas, child, drawingTime);
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/conversationpager/SlidingPaneLayout.java]: Want to remove the unused imports from here as well? 
```
android.graphics.Bitmap
java.lang.reflect.Field
java.lang.reflect.Method
```

File: app/src/main/java/com/waz/zclient/pages/main/participants/OptionsMenuFragment.java [hunk 2]
```diff
@@ -238,7 +238,6 @@ public void onMenuConversationHasChanged(final IConversation conversation) {
                 items.add(OptionsMenuItem.PICTURE);
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/participants/OptionsMenuFragment.java]: Care to add a `break` before `case WAIT_FOR_CONNECTION` as well? I guess someone must have missed that one.

**Comment by hakonbo** [comment]
build again

**Comment by hakonbo** [comment]
Great, thanks @pprucia

**Pull request merged**